WebApiCoreSeed.WebApi/Controllers/ValuesController.cs

```diff
@@ -1,45 +1,53 @@
-﻿using System;
+
+namespace WebApiCoreSeed.WebApi.Controllers
+{
+
+using System;

```

Comment:
When `using` statements are inside the namespace, it's common to indent them like:

```C#
namespace WebApiCoreSeed.WebApi.Controllers
{
    using System;
...
```